test/e2e/app-dir/metadata-icons-parallel-routes/metadata-icons-parallel-routes.test.ts

```diff
@@ -11,4 +11,14 @@ describe('app-dir - metadata-icons-parallel-routes', () => {
     expect($('link[type="image/svg+xml"]').length).toBe(1)
     expect($('link[rel="apple-touch-icon"]').length).toBe(1)
   })
+
+  it('should override parent icon when both static icon presented', async () => {
+    const $ = await next.render$('/nested')
+    expect($('link[rel="icon"][type="image/png"]').length).toBe(1)
```

Comment:
maybe just add an assertion here to ensure only 1 favicon is rendered to confirm that what i was seeing was some weird cache issue

Reply:
Added, we already had sth for that case in another test tho
https://github.com/vercel/next.js/blob/4595c5de8858e57c2bb1ebf5158b1d607242c297/test/e2e/app-dir/metadata-icons/metadata-icons.test.ts#L15